master/de.po

```diff
@@ -21486,15 +21512,17 @@ msgstr ""
 "Null setzen, um Cachen zu deaktivieren"
 
 #: ../src/ui/dialog/inkscape-preferences.cpp:1473
-#, fuzzy
 msgid "Rendering tile multiplier:"
-msgstr "Rendering-Cachegröße:"
+msgstr "Renderkachel-Multiplikator:"
```

Comment:
Das ist zwar wahrscheinlich die direkteste Übersetzung für "Rendering tile multiplier" aber ich glaube damit kann erstmal niemand etwas anfangen.

Was hältst du von "Kachelgröße für Renderer"? Passt nicht genau (weil der Multiplikator rausfällt), da jedoch völlig unklar ist, was hier überhaupt multipliziert wird (und wie), ist das vielleicht sogar eher ein Vorteil.

Reply:
Ja, keine Ahnung was das tut... Und ob der Wert nun proportional oder umgekehrt proportional zur Kachelgröße ist... Wäre echt toll, wenn jemand dazu was sagen könnte... Und vor allem, den Tooltip um nützliche Inhalte erweitern könnte, wie z.B. 'was bewirkt eine Änderung dieses Wertes in Bezug auf meine Arbeit mit Inkscape' - der aktuelle Tooltip ist, pardon, einfach Schrott. Was soll ein Nutzer damit anfangen? Ich kann die Übersetzung hier nur verbessern, wenn ich weiß, was es tut.

Reply:
Oh, Moment. "Je größer der Wert, desto größer sind die einzelnen Kacheln." - immerhin. Da passt Dein Vorschlag ja einigermaßen.

Reply:
Wenn ich es richtig verstehe wird die Leinwand in Rechtecke mit maximal 16384 Pixeln (z.B. 128x128) unterteilt und diese separat gerendert. Das verhindert, dass die Berechnung zu lange dauert (und z.B. die UI blockiert), kann aber ineffizienter sein als größere Rechtecke am Stück zu rendern.  Der "Rendering tile multiplier" vergrößert diese maximale Anzahl an Pixeln entsprechend.

> 'was bewirkt eine Änderung dieses Wertes in Bezug auf meine Arbeit mit Inkscape' 

Ich befürchte diese Frage hat nie jemand wirklich beantwortet...  Tav hat angewmerkt, dass auf moderner Hardware ein größerer Wert meist vorteilhaft ist, wie weit man das aber treiben kann und ob es einen Optimalwert gibt wollte niemand austesten...

Reply:
Mmh, okay. Vielleicht könnten wir dann für master (von inkscape) lieber irgendwas anderes schreiben...?

Sowas wie 'On modern hardware, increasing this value (default: 16) can lead to improved performance with faster redrawing of the canvas.' Selbst wenn das immer noch nicht so richtig klar ist, ergibt das Vorhandensein der Einstellung dann wenigstens einen Sinn, und Experimentierfreudige wissen, worauf sie achten müssen. Soll ich's probieren und Tav taggen?

Reply:
Ja, immer noch schwammig, aber solange niemand systematisch testet wird's wohl nicht besser... Vielleicht kan Tav noch was beisteuern, sonst bleibt's halt eine "trial-and-error" Funktion.

Reply:
Hab mich mal dran versucht. Schau mer mal. 
https://gitlab.com/inkscape/inkscape/merge_requests/211

Reply:
Super, danke!